### PR TITLE
dash/docs-sync-charts

### PR DESCRIPTION
**Pull request by sebastianbochan**
Added docs about component's sync.

### CODE REVIEW

File: docs/dashboards/synchronize-components.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+Groups which have shared state can be accessible via
+```js
+component.activeGroup.getSharedState()
+```
+
+Also you can post data between components in the same group via `postMessage` method.
+```js
+component.postMessage('hello world');
+```
```
Comment (pawellysy) [inline docs/dashboards/synchronize-components.md:100]: I'm wondering if we need this part here. First of all the groups are not implemented (or are they?) and recently we deleted the ability to postMessage. We can add this part later, when the feature will be implemented. No need to confuse users

**Review by pawellysy — CHANGES_REQUESTED** [review]

File: docs/dashboards/synchronize-components.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+Synchronizing components
+===
+
+Components might need to work together to accomplish a specific task.
```
Comment (karolkolodziej) [inline docs/dashboards/synchronize-components.md:4]: > might need to work together to accomplish a specific task.

I'm not sure if that is the best set of words to describe sync. For me, it's sth extra that helps visualize, navigate and highlight some of the data.

File: docs/dashboards/synchronize-components.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+<iframe style="width: 100%; height: 470px; border: none;" src=https://www.highcharts.com/samples/embed/dashboards/demo/minimal allow="fullscreen"></iframe>
+
+## How to synchronize the components?
+To synchronize components you have to specify which event you want to synchronize between each component, as well as they have to use the same connector defined in the [dataPool](https://www.highcharts.com/docs/dashboards/data-handling).
+
+The events, that can be synchronized between components are:
+* [visibility](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/component-options/sync-visibility/)
+* [extremes](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/demo/sync-extremes/)
+* [highlight](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/component-options/sync-highlight/)
+
```
Comment (karolkolodziej) [inline docs/dashboards/synchronize-components.md:17]: Please add a table where each component will be listed with all supported sync

File: docs/dashboards/synchronize-components.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+Groups which have shared state can be accessible via
+```js
+component.activeGroup.getSharedState()
+```
+
+Also you can post data between components in the same group via `postMessage` method.
+```js
+component.postMessage('hello world');
```
Comment (karolkolodziej) [inline docs/dashboards/synchronize-components.md:99]: the `postMessage` should not be described. we removed this functionality

File: docs/dashboards/synchronize-components.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+      type: 'Highcharts',
+      columnAssignment: {
+          Food: 'x',
+          'Vitamin A': 'y'
+      }
+  }]
+});
+```
+
+## Component groups
```
Comment (karolkolodziej) [inline docs/dashboards/synchronize-components.md:78]: I'm not sure we should describe this. let's discuss it internally.

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
Please describe custom sync based on: `dashboards/demo/datacursor-sync`

File: docs/dashboards/synchronize-components.md (hunk 1 above)
Comment (sebastianbochan) [reply docs/dashboards/synchronize-components.md:100]: So we need to remove it from other parts too.

**Review by Vazonik — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]

**Pull request merged**